packages/core/src/App/Containers/AccountSignupModal/account-signup-modal.jsx

```diff
@@ -113,6 +120,51 @@ const AccountSignup = ({
         setABQuestionnaire(fetchQuestionnarieData());
     }, []); // eslint-disable-line react-hooks/exhaustive-deps
 
+    const trackSignupErrorEvent = (action, errorMessage, screen_name) => {
+        const form_name = is_mobile ? 'virtual_signup_web_mobile_default' : 'virtual_signup_web_desktop_default';
+        cacheTrackEvents.loadEvent([
+            {
+                event: {
+                    name: 'ce_virtual_signup_form',
+                    properties: {
+                        action,
+                        form_name,
+                        error_message: errorMessage,
+                        screen_name,
+                    },
+                },
+                cache: true,
```

Comment:
Do we always need it to be cached? It can be sent directly if rudderstack is available.

Reply:
Actually country screen is the first screen that loads and when in 3g or slow networks and when an console throws an error,we should cache it.If not we will miss out some of the console errors in slow networks.

Reply:
Okay `cache: true`is removed